src/Features.js

```diff
@@ -724,6 +725,23 @@ function Features({ history }) {
                     )}
                   </NavLink>
                 </NavItem>
+                <NavItem>
+                  <NavLink
+                    className={getTabClassName('clinical_features')}
+                    onClick={() => {
+                      toggle('clinical_features');
+                    }}
+                  >
+                    {getTabSymbol()}
+                    {isAlternativeUser ? (
```

Comment:
This test for an "alternative" user is not needed here, and might be obsolete overall (I will look into this in the future). There was initially an idea to have 2 version of the platform, one that includes visualization of features and one without, to make comparisons and see the benefit of visualizing the features. This idea wasn't realized in the end, and this test only applied to the Visualization tab in any case, as it needed to be called differently depending on the user group.